src/OrchardCore/OrchardCore.Contents.Core/SiteServiceExtensions.cs

```diff
@@ -0,0 +1,17 @@
+using System.Threading.Tasks;
+using OrchardCore.ContentManagement;
+using OrchardCore.Entities;
+
+namespace OrchardCore.Settings;
+
+public static class SiteServiceExtensions
+{
+    /// <summary>
+    /// Gets an instance of the specified settings if it exists.
+    /// </summary>
+    /// <param name="siteService">The site service.</param>
+    /// <param name="name">The entry name.</param>
+    /// <returns>An <see cref="ContentItem" /> instance that matches the given name, if one exists.</returns>
```

Comment:
Adding a `<param>` for the `this` parameter is not really useful, since it'll never show up in IntelliSense. Same for the other `GetSettingsAsync`.

```suggestion
    /// <summary>
    /// Gets an instance of the specified custom settings if it exists.
    /// </summary>
    /// <param name="name">The entry name.</param>
    /// <returns>A <see cref="ContentItem" /> instance that matches the given name, if one exists.</returns>
```

Reply:
this broken the build :). I'll add it back up

Reply:
Eh. Please keep the other changes.